src/main/resources/liquibase/jpcontentscheduler/port/00000000000001_dataPort_test.sql

```diff
@@ -1,94 +1,94 @@
-         <!--categorie associate al tipo di contenuto e che verranno associate ai contenuti in fase di depubblicazione o sospensione 
-                nell attributo code deve essere specificato il codice della categoria-->
-		<!--<category code="pubb_drt" />
-		<category code="cnol_canale8" /> -->
-	 </contentType>
-	<contentType type="SNT" startAttr="Data_inizio" endAttr="Data_fine" suspend="true" >
-        <!--non essendoci categorie associate, ai contenuti di questo tipo verrà associata la categoria globale -->
-	 </contentType>
-</contentTypes>
-
-<!-- i gruppi attualmente non vengono usati-->
-<groups> 
-          <group id="gruppoDiProvaThread" contentType="NOL" /> 
-</groups>
-  
-<!-- utenti che riceveranno le comunicazioni sull andamento delle operazioni di pubblicazione, sospensione e spostamento -->
-<users>
-<!-- l attributo username deve contenere lo username dell utente
- l attributo contentType deve contentere la tipologia o le tipologie di contenuto separate da virgola per cui ricevere le notifiche 
-<user username="username1" contentType="SNT,DRT,tipo3...." />
-Se contentType="*" verranno inviate le notifiche per tutti i tipi di contenuti presenti nella configurazione
- -->
-<user username="username1" contentType="*" />
-<user username="username2" contentType="EVN" />
-<user username="username3" contentType="*" />
-</users>
-
-<!-- template della mail di comunicazione-->
-<mail alsoHtml="false" senderCode="CODE1" mailAttrName="email" > 		
-<subject><![CDATA[Report pubblicazione/sospensione automatica]]></subject> 		 
- <htmlHeader><![CDATA[<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"> <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="it">  <head> </head>  <body><div style="padding:1.5em;font-family:Arial, sans serif; color: #333333;font-size:0.9em">Report  Pubblicazione/Sospensione automatica contenuti<br/><p>Contenuti pubblicati:</p>]]></htmlHeader> 
- 		 <htmlFooter><![CDATA[<br /><br />Cordiali Saluti<br/></div></body></html>]]> </htmlFooter> 		  
-<htmlSeparator> <![CDATA[ <br /> Contenuti sospesi: ]]> </htmlSeparator> 		 
- <textHeader><![CDATA[Report pubblicazione/sospensione automatica]]> </textHeader>  
-<textFooter><![CDATA[   Cordiali Saluti.]]> </textFooter> 	 	 
-<textSeparator><![CDATA[
-
-Contenuti sospesi:
- ]]></textSeparator> 	
-
-<htmlSeparatorMove> <![CDATA[ <br /> Contenuti spostati in archivio online: ]]> </htmlSeparatorMove> 
- <textHeaderMove><![CDATA[Report pubblicazione/sospensione automatica]]> </textHeaderMove>  
-<textFooterMove><![CDATA[   Cordiali Saluti.]]> </textFooterMove>	 
-<textSeparatorMove><![CDATA[
-
-Contenuti spostati in archivio online:
- ]]></textSeparatorMove> 	 
-
+INSERT INTO sysconfig (version, item, descr, config) VALUES ('test', 'cthread_config', 'Configurazione thread pubblicazione/sospensione automatica', '<contentThreadconfig sitecode="A">
+<!--attiva o disattiva lo scheduler.
```

Comment:
This value is longer than 4000 characters and will cause issues with Oracle, so this insert should be converted in XML format and the inserted text moved to a separate file.

Reply:
Done